app/hedge.py

```diff
@@ -164,8 +166,19 @@ def main():
         time_taken = end_time - start_time
         if time_taken < cycle_time:
             time.sleep(cycle_time - time_taken)
+        return None
+
+    elif last_ran != datetime.now().strftime('%H'):
+        print(str(last_ran) + '          ' + datetime.now().strftime('%H'))
+        last_ran = datetime.now().strftime('%H')
+        sftp.pull_videos('10.170.1.11', 'pi', 'hog1hog1')
```

Comment:
I would set the ip address, username, and password, as constants at the top of this file

i.e. 
```
PIZERO_IP = '10.179.1.11'
PIZERO_FTP_USERNAME = `pi`
PIZERO_FTP_PASSWORD = `hog1hog1`
```

In terms of security, we should not really have passwords committed in code at all. If we are confident that our pizero is never accessible from the internet, we could either (a) add a comment to that effect next to the password definition, or (b) set up the FTP server to allow anonymous connections.

Reply:
I agree, maybe use system variables or something